src/app/models/like.model.ts

```diff
@@ -0,0 +1,4 @@
+export interface Like {
+  id?: number;
+  uid?: string;
```

Comment:
some docs might be helpful here
```suggestion
  /** id of the liked song in genius */
  id?: number;
  /** firebase id of the user who liked the song */
  uid?: string;
```